module/product/src/Resources/translations/notification.en.yml

```diff
@@ -0,0 +1,2 @@
+Batch action "%type%" ended: Batch action "%type%" ended
```

Comment:
First translation should be in batch action module becous this is universal functionality 
Second line is connected to product delete batch action and that schold be in product module